src/commands/set.js

```diff
@@ -20,16 +20,21 @@ import { writeJSONSync } from '../utils/fs';
 import { createCommand } from '../utils/helpers';
 import liskAPIInstance from '../utils/api';
 
-const description = `Sets configuration <variable> to <value>. Variables available: json, name, testnet. Configuration is persisted in \`${configFilePath}\`.
+const availableVariables = CONFIG_VARIABLES.join(', ');
+const description = `Sets configuration <variable> to <value>. Variables available: ${availableVariables}. Configuration is persisted in \`${configFilePath}\`.
 
 	Examples:
 	- set json true
 	- set name my_custom_lisky
+	- set api.testnet true
 `;
 
 const WRITE_FAIL_WARNING =
 	'Config file could not be written: your changes will not be persisted.';
 
+const writePropertyWarning = property =>
```

Comment:
I see that the naming here conforms to `WRITE_FAIL_WARNING`, but this is a confusing name for a function as nothing's being written. The function's only used once, is it needed?